core/src/main/java/org/apache/sdap/mudrod/ssearch/ranking/DataGenerator.java

```diff
@@ -34,10 +33,10 @@
   private static boolean isMultFiles;
 
   private static String[] myHeader;
-  private static List<List<String>> myMasterList = new ArrayList<>();
+  private static List<List<String>> myMasterList = new ArrayList<List<String>>();
 
   // HashMap used for comparing evaluation classes
-  public static final Map<String, Integer> map1 = new HashMap<>();
+  public static final HashMap<String, Integer> map1 = new HashMap<String, Integer>();
```

Comment:
Return types should not be concrete implementations if possible.  Also use diamond operator